regolith/helpers/a_presentationhelper.py

```diff
@@ -190,7 +188,7 @@ def db_updater(self):
                      'begin_date': begin_date,
                      'end_date': end_date,
                      })
-        if rc.notes:
+        if rc.notes is not None:
```

Comment:
ok, I change my mind.  I was thinking it was updating the notes but it is creating a presentation so the original construction is better.  You were right before.  Sorry for messing you up there....

One thing to check is how the parser works if only one note is given, does it return a string or a list with one entry, i.e., does `--notes "one note"` result in `rc.notes` being `"one note"` or ["one note"].  Please check, and make sure we pass a list of strings to the `update`

Reply:
No worries. Reverting changes now and will push to a new branch. If --notes "one note" is given, the result is "one note." If --notes "one note" "two note", the result is "one note " "two note." To clarify, you want rc.notes to format the strings in the notes section as a list of strings, i.e., ["one note", "two note"]?

Reply:
> No worries. Reverting changes now and will push to a new branch. If --notes "one note" is given, the result is "one note." If --notes "one note" "two note", the result is "one note " "two note." To clarify, you want rc.notes to format the strings in the notes section as a list of strings, i.e., ["one note", "two note"]?

"one note" "two note" is not a type so it can't return that.  It probably returns ["one note", "two note"].  You can check by asking what type is returned, a list or a string?  I think I would like "notes" everywhere in the program to have the same behavior and to take a list of strings, so in the end we would want ["one note", "two note"], or ["one note"] but not "one note".  It just makes things easier.   we may want to make a new issue to check in the schema that this is required everywhere, and then update it everywhere in the code.

Reply:
for now, just handle it this way in this case.   Is there a notes field in the schema for `presentations`?  Do we want one?  It should be there for expenses, for sure.  It is sometimes better to put it in as an empty list even if it is not required in the schema since it doesn't really cost too much to do it and then someone can more easily add a note later.   But only if it is there in schema in the first place (btw, the schema is in `schema.py` in `regolith/regolith`)